cosmos/xion-testnet.json

```diff
@@ -0,0 +1,50 @@
+{
+  "rpc": "https://rpc.xion-testnet-1.burnt.com/",
+  "rest": "https://api.xion-testnet-1.burnt.com/",
+  "chainId": "xion-testnet-1",
+  "chainName": "Xion Testnet",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/xion-testnet/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "XION",
+    "coinMinimalDenom": "uxion",
+    "coinDecimals": 6,
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/xion-testnet/chain.png"
+  },
+  "walletUrl": "https://wallet.keplr.app/chains/xion-testnet",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/xion-testnet",
```

Comment:
The page isn't supported by Keplr Dashboard dynamically.
You can remove these two properties from the change.

Reply:
Done